opencti-platform/opencti-front/src/components/InputSliderField.tsx

```diff
@@ -0,0 +1,127 @@
+import { Field, FieldProps } from 'formik';
+import React, { FunctionComponent } from 'react';
+import { Slider } from '@mui/material';
+import TextField from './TextField';
+import { SubscriptionFocus } from './Subscription';
+import { useLevel } from '../utils/hooks/useScale';
+import { Option } from '../private/components/common/form/ReferenceField';
+import { useFormatter } from './i18n';
+
+interface InputSliderFieldProps {
+  label: string;
+  variant?: string;
+  onSubmit?: (name: string, value: string | number | number[] | Option) => void;
+  onFocus?: (name: string, value: string) => void;
+  editContext?: unknown;
+  containerStyle?: Record<string, string | number>;
+  entityType: string;
+  attributeName: string;
+  disabled?: boolean;
+}
+
+const InputSliderField: FunctionComponent<InputSliderFieldProps & FieldProps> = ({
+  form: { setFieldValue },
+  field: { name, value },
+  label,
+  variant,
+  onFocus,
+  onSubmit,
+  containerStyle,
+  editContext,
+  entityType,
+  attributeName,
+  disabled,
+}) => {
+  const { level: { color }, marks } = useLevel(entityType, attributeName, value);
+  const { t } = useFormatter();
+
+  const min = marks.length > 0 ? marks[0].value : 0;
+  const max = marks.length > 0 ? marks[marks.length - 1].value : 0;
+
+  const sliderMarks = marks.map((mark) => ({
+    value: mark.value,
+    label: mark.label ? t(mark.label) : mark.value,
```

Comment:
t(mark.label ?? mark.value)

Reply:
No, I just want the code to be simplier. The formatter won't translate something if is not a given key so you can pass him anything you don't have to make conditionnal